### PR TITLE
Provide concept guide for the Kanto Local Digital Twins

### PR DESCRIPTION
**Pull request by antonia-avramova**
[#43] Provide concept guide for the Kanto Local Digital Twins

Signed-off-by: Antonia Avramova <antonia.avramova@bosch.io>

### CODE REVIEW

File: web/site/content/docs/concepts/local-digital-twins.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: "Local digital twins"
+type: docs
+description: >
+  Empower the edge device with local digital twins for more advanced offline scenarios.
+weight: 5
+---
+
+Local digital twins enables the digital twin state on a local level even in offline scenarios. The following use cases are provided:
```
Comment (k-gostev) [inline web/site/content/docs/concepts/local-digital-twins.md:9]:
```suggestion
Local digital twins enables the digital twin state on a local level even in offline scenarios. It provides the following use cases:
```

File: web/site/content/docs/concepts/local-digital-twins.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+* **Mirrors the applications**
+* **Provides diverse services for the applications**
+* **Provides own persistency** - digital twins are stored locally
+* **Connectivity** - provide connectivity to the cloud (similar to [Suite connector]({{< relref "suite-connector" >}}))
+* **Offline scenarios** - local applications stay fully operable as if the connection with the cloud is not interrupted
+* **Synchronization** - on a connectivity recovering synchronizes the last known digital twin state
```
Comment (k-gostev) [inline web/site/content/docs/concepts/local-digital-twins.md:16]:
```suggestion
* **Synchronization** - when connection to the cloud is established  the last known digital twin state is synchronized
```

File: web/site/content/docs/concepts/local-digital-twins.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  Empower the edge device with local digital twins for more advanced offline scenarios.
+weight: 5
+---
+
+Local digital twins enables the digital twin state on a local level even in offline scenarios. The following use cases are provided:
+
+* **Mirrors the applications**
+* **Provides diverse services for the applications**
+* **Provides own persistency** - digital twins are stored locally
+* **Connectivity** - provide connectivity to the cloud (similar to [Suite connector]({{< relref "suite-connector" >}}))
```
Comment (k-gostev) [inline web/site/content/docs/concepts/local-digital-twins.md:14]:
```suggestion
* **Cloud connectivity** - similar to [Suite connector]({{< relref "suite-connector" >}})
```

File: web/site/content/docs/concepts/local-digital-twins.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+type: docs
+description: >
+  Empower the edge device with local digital twins for more advanced offline scenarios.
+weight: 5
+---
+
+Local digital twins enables the digital twin state on a local level even in offline scenarios. The following use cases are provided:
+
+* **Mirrors the applications**
+* **Provides diverse services for the applications**
```
Comment (k-gostev) [inline web/site/content/docs/concepts/local-digital-twins.md:12]:
```suggestion
* **Diverse services for the applications**
```

File: web/site/content/docs/concepts/local-digital-twins.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+* **Provides own persistency** - digital twins are stored locally
```
Comment (k-gostev) [inline web/site/content/docs/concepts/local-digital-twins.md:13]:
```suggestion
* **Persistency** - digital twins are stored locally
```

File: web/site/content/docs/concepts/local-digital-twins.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+* **Connectivity** - provide connectivity to the cloud (similar to [Suite connector]({{< relref "suite-connector" >}}))
+* **Offline scenarios** - local applications stay fully operable as if the connection with the cloud is not interrupted
+* **Synchronization** - on a connectivity recovering synchronizes the last known digital twin state
+
+![Local Digital Twins](/kanto/images/docs/concepts/local-digital-twins.png)
+
+### How it works
```
Comment (k-gostev) [inline web/site/content/docs/concepts/local-digital-twins.md:20]: TODO @k-gostev - this whole paragraph needs rewriting

**Review by k-gostev — CHANGES_REQUESTED** [review]

File: web/site/content/docs/concepts/local-digital-twins.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+weight: 6
+---
+
+Local digital twins enables the digital twin state on a local level even in offline scenarios. It provides the following use cases:
+
+* **Mirrors the applications**
+* **Persistency** - digital twins are stored locally
+* **Cloud connectivity** - provide connectivity to the cloud (similar to [Suite connector]({{< relref "suite-connector" >}}))
+* **Offline scenarios** - local applications stay fully operable as if the connection with the cloud is not interrupted
+* **Synchronization** - when connection to the cloud is established  the last known digital twin state is synchronized
```
Comment (dimitar-dimitrow) [inline web/site/content/docs/concepts/local-digital-twins.md:15]:
```suggestion
* **Synchronization** - when connection to the cloud is established the last known digital twin state is synchronized
```

File: web/site/content/docs/concepts/local-digital-twins.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+
+![Local Digital Twins](/kanto/images/docs/concepts/local-digital-twins.png)
+
+### How it works
+
+Similar to the [Suite connector]({{< relref "suite-connector" >}}) service the **local digital twins** service needs to establish a connection to the cloud. To do this this the edge has to be manually or automatically provisioned. This connection is then used as a channel to pass the edge telemetry and event messages. Once a connection is established the device can be operated via commands.
+
+To ensure that the digital twin state is available on a local level even in offline mode (no matter of the connection state) the **local digital twins** service persist all changes locally. Such capabilities were implemented to support offline scenarios and advanced edge computing involving synchronization with the cloud after disruptions or outages. The synchronization mechanisms were also designed in a way to significantly reduce data traffic, and efficiently prevent data loss due to long-lasting disruptions.
+
+Upon reconnection the **local digital twins** it will notifies the cloud of any changes since during the offline mode to to synchronize the digital twins state.
```
Comment (dimitar-dimitrow) [inline web/site/content/docs/concepts/local-digital-twins.md:25]:
```suggestion
Upon reconnection, the **local digital twins** will notify the cloud of any changes during the offline mode and synchronize the digital twins state.
```

**Pull request merged**